src/io/flutter/logging/FlutterLogEntryParser.java

```diff
@@ -126,47 +146,53 @@ private FlutterLogEntry parseLoggingEvent(@NotNull Event event) {
       }
     }
 
+    final FlutterDebugProcess debugProcess = getDebugProcess();
+
     final String messageStr = message.getValueAsString();
     final FlutterLogEntry entry = lineHandler.parseEntry(messageStr, category, level);
 
     if (message.getValueAsStringIsTruncated()) {
       entry.setMessage(messageStr + "...");
-      final Isolate isolate = new Isolate(json.get("isolate").getAsJsonObject());
-      debugProcess.getVmServiceWrapper().getObject(isolate.getId(), message.getId(), new GetObjectAdapter() {
-        @Override
-        public void received(Obj response) {
-          entry.setMessage(((Instance)response).getValueAsString());
-          contentChangedDispatcher.getMulticaster().onContentUpdate();
-        }
-      });
+      if (debugProcess != null) {
+        final Isolate isolate = new Isolate(json.get("isolate").getAsJsonObject());
+        debugProcess.getVmServiceWrapper().getObject(isolate.getId(), message.getId(), new GetObjectAdapter() {
+          @Override
+          public void received(Obj response) {
+            entry.setMessage(((Instance)response).getValueAsString());
```

Comment:
not a new issue but isn't this on a background thread updating UI code?

Reply:
On my list to look into.  Thanks for the nudge.